taskcluster/docker/csharp/Dockerfile

```diff
@@ -0,0 +1,117 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster
+
+LABEL maintainer="aplacitelli@mozilla.com"
+
+RUN echo "trigger rebuild"
```

Comment:
Let's actually get rid of this :D 
That's a leftover when I copied over the other Dockerfile. It was used sometime ago in a-s repo to trigger a rebuild of the image. As this is the first time it's build we don't need to have it.